src/app/components/itinerary/itineraryshow/itineraryview.component.ts

```diff
@@ -0,0 +1,73 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7181090)
```suggestion
import { Component, OnInit } from "@angular/core";
```